Keyboards/KeyboardsBase/ToolTip/Model/InformationToolTipData.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  InformationToolTipData.swift
+//  Scribe
+//
+//  Created by Gabriel Moraes on 03/12/22.
+//
+
+import Foundation
+
+enum InformationToolTipData {
+  
+  static let wikiDataExplanation = "Wikidata is a collaboratively edited knowledge graph that's maintained by the Wikimedia Foundation. It serves as a source of open data for projects like Wikipedia."
```

Comment:
Being a bit lazy here and I don't want to find exactly where the text width is set. Could we shrink the `textView` with for these texts to 90% of the width of the tooltip? I think they're a bit too close to the edge.